frontend/src/metabase/query_builder/components/QueryModals.jsx

```diff
@@ -237,3 +257,21 @@ export default class QueryModals extends React.Component {
     ) : null;
   }
 }
+
+const QuestionMoveToast = ({ question, collectionId }) => {
+  const isModel = question.isDataset();
+  return (
+    <ToastRoot>
+      <Icon name="all" mr={1} color="white" />
+      {isModel
+        ? jt`Model moved to ${(
```

Comment:
Not sure if it's better to use
```suggestion
        ? jt`${entity} moved to ${(
```

when entity could be either **Model** or **Dashboard**. I assume by doing what I've done here would make the translation easier. @ranquild what do you think?

Reply:
It's easier for translators to have full messages, e.g. `Model moved to xxx` because some languages require changes to the noun in that case.